### PR TITLE
spring-cloud: add support to enable/disable MSI (#1523)

### PR DESCRIPTION
**Pull request by leonard520**
Add command "az spring-cloud app identity" to support Managed Identity feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yungezz** [comment]
hi @leonard520 is there any change between this PR with previous PR? that one was reviewed and approved already

**Comment by leonard520** [comment]
> hi @leonard520 is there any change between this PR with previous PR? that one was reviewed and approved already

They are the same except I raise the version number to 0.2.4 in this one

**Comment by yungezz** [comment]
thanks for clarification. Then just ping us when the PR is ready to merge, thanks. @leonard520

**Comment by leonard520** [comment]
@yungezz @fengzhou-msft The backend is ready. Please help to merge the code. Thanks!

File: src/spring-cloud/HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Release History
 ===============
 
+0.2.4
+-----
+* Add command "az spring-cloud app identity"  to support Managed Identity feature
```
Comment (qwordy) [inline src/spring-cloud/HISTORY.md:6]:
```suggestion
* Add command "az spring-cloud app identity" to support Managed Identity feature
```

**Comment by qwordy** [comment]
Could you please add a brief description for this PR and link the previous same PR that is reviewed?

**Comment by leonard520** [comment]
#1671 https://github.com/Azure/azure-cli-extensions/pull/1523

File: src/spring-cloud/HISTORY.md (hunk 1 above)
Comment (qwordy) [inline src/spring-cloud/HISTORY.md:6]:
```suggestion
* Add command "az spring-cloud app identity" to support Managed Identity feature
```

**Pull request merged**